maxim/logger/agno/client.py

```
@@ -0,0 +1,188 @@
+        self,
+        *args: Any,
+        trace_id: Optional[str] = None,
+        generation_name: Optional[str] = None,
+        **kwargs: Any,
+    ) -> Any:
+        trace, generation, _, end_trace = _start_trace(logger, self, trace_id, generation_name)
+        result = None
+        try:
+            result = fn(self, *args, **kwargs)
+            if inspect.isgenerator(result):
+                def _iterate() -> Any:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `_iterate`

(ANN401)